CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.7.1 [2018.12.14]
```

Comment:
Let's change this to `## next [????.??.??]`, since (1) I definitely won't be releasing a new version of this library on that day (it's passed), and (2) it's yet unclear if I'll be giving it that version number.